api/v1beta2/ibmvpccluster_types.go

```diff
@@ -94,6 +98,24 @@ type AdditionalListenerSpec struct {
 	Port int64 `json:"port"`
 }
 
+// VPCNetworkSpec defines the desired state of the network resources for the cluster.
+type VPCNetworkSpec struct {
+	// computeSubnetsSpec is a set of Subnet's which define the Compute subnets.
+	// +optional
+	ComputeSubnetsSpec []Subnet `json:"computeSubnetsSpec,omitempty"`
```

Comment:
if it is for workers, can it be more specific like workerSubnet or something more meaningful, because every node is a compute in the cluster and please remove the `Spec` from the end from all the fields

Reply:
Updated